### ISSUE TITLE
Wrong values total production /consumption

### ISSUE THREAD

**Issue by frommelt5** [vikt0rm/dbus-shelly-1pm-pvinverter#1]
Hey, 
This piece if software is great but in my setup there is an issue with the total production and consumption values. Every morning at around 6 o'clock it is adding a huge amount of kWh, probably the value of the hole  last day.
The other values are always correct.
Setup: 2 Shelly 1pm on Hoymiles Inverters and Victron multiplus II (ESS Mode).
Any idea? What did I misconfigure? I think the values were correct before I started using this script 
![Screenshot_20220418-160202_VRM](https://user-images.githubusercontent.com/14992447/163819524-ab16b815-efb1-4dc8-a43f-ddcac728f386.jpg)

**Comment by vikt0rm** [comment]
Hi, 

yes i saw that issue as well. Need to take a closer look what exactly is going wrong there. I will fix that hopefully today. 
Thank you for the feedback!

BR,
Viktor

**Comment by vikt0rm** [comment]
Issue is fixed with last [commit](https://github.com/vikt0rm/dbus-shelly-1pm-pvinverter/commit/39c38ce6392f7f2f40410d040254113930a17d03) on master.

**Comment by frommelt5** [comment]
I thought it was the solution, seemed to work initially. But yesterday and today the same bug again. I tried a reinstall, didn't help. Is it working for you?

**Comment by vikt0rm** [comment]
Yes it worked some days without issues, so I thought its good to go. But yesterday I also had a use case where energy value again was 0 causing this issue (root cause). 

I try to invalidate the initial energy values and see if that helps.
